bazel/external/quiche.BUILD

```diff
@@ -396,10 +511,22 @@ envoy_cc_test(
     deps = [":spdy_platform"],
 )
 
+envoy_cc_test(
+    name = "spdy_header_block_test",
```

Comment:
nit: Rename to spdy_core_header_block_test?

Reply:
done